webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx

```diff
@@ -158,10 +114,33 @@ class RhsCardHeader extends React.PureComponent<Props> {
                     />
                 </span>
                 <div className='pull-right'>
-                    <OverlayTrigger
-                        delayShow={Constants.OVERLAY_TIME_DELAY}
+                    <WithTooltip
+                        id={this.props.isExpanded ? 'shrinkSidebarTooltip' : 'expandSidebarTooltip'}
+                        title={this.props.isExpanded ?
+                            <>
+                                <FormattedMessage
+                                    id='rhs_header.collapseSidebarTooltip'
+                                    defaultMessage='Collapse the right sidebar'
+                                />
+                                <KeyboardShortcutSequence
+                                    shortcut={KEYBOARD_SHORTCUTS.navExpandSidebar}
+                                    hideDescription={true}
+                                    isInsideTooltip={true}
```

Comment:
Have you verified is there a UI change before and after? Can you provide screenshots

Reply:
previous
![image](https://github.com/user-attachments/assets/b7c9b964-724e-4df7-bfd2-98a656df3e49)
|
After
![image](https://github.com/user-attachments/assets/c99f1547-9ebb-4c46-b3ee-0aff0959dcc5)

Here are the screenshots from before and after I changed the tooltips